.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
```

Comment:
Consider adding `preLaunchTask` or `postDebugTask` to handle any setup or teardown tasks needed during the debugging process.